address/src/main/java/com/alibaba/nacos/address/config/NacosSecurityConfiguration.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 1999-2022 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.address.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+/**
+ * nacos web security configuration.
+ * @author onewe
+ */
+@Configuration
+public class NacosSecurityConfiguration extends WebSecurityConfigurerAdapter {
+    
+    @Override
+    public void configure(WebSecurity web) {
+        web.ignoring().mvcMatchers(HttpMethod.GET, "/*/*");
```

Comment:
Need some unit tests.

`/*/*` may contains `/v1/as/...`?

Reply:
it doesn't contain `/v1/as/..`, I have already tested it.